src/main/java/org/dbflute/intro/app/web/playsql/data/PlaysqlDataAction.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2014-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+package org.dbflute.intro.app.web.playsql.data;
+
+import javax.annotation.Resource;
+
+import org.dbflute.intro.app.logic.playsql.data.PlaysqlDataLogic;
+import org.dbflute.intro.app.web.base.IntroBaseAction;
+import org.lastaflute.web.Execute;
+import org.lastaflute.web.response.JsonResponse;
+
+/**
+ * @author prprmurakami
+ */
+public class PlaysqlDataAction extends IntroBaseAction {
+
+    // ===================================================================================
+    //                                                                           Attribute
+    //                                                                           =========
+    @Resource
+    private PlaysqlDataLogic playsqlDataLogic;
+
```

Comment:
Insert the tag comment "Execute" here by "_taexec" completion.

Reply:
@jflute 
Fixed.
20ca0e474c10584f8a2f510f6c73791df4008c8a